lib/home_page.dart

```diff
@@ -514,6 +242,349 @@ class HomeState extends State<Home> with WidgetsBindingObserver {
     );
   }
 
+  Widget _build(
+    BuildContext context,
+    BreezUserModel user,
+    AccountModel account,
+    AccountSettings settings,
+    LSPStatus lspStatus,
+    List<AddFundVendorModel> vendor,
+  ) {
+    final addFundsVendors = _drawerConfigAddFundsVendors(
```

Comment:
It seems this drawer piece is not being used anymore, I kept it but I guess it can be removed.